lib/services/pubsub/service.js

```diff
@@ -771,7 +794,7 @@ PubSubService.prototype.publish = function (message, payload) {
   opts.hasWildcard = true; // all remaining emit attempts are to wildcard subscribers
 
   for (var allPath in this.__listeners_wildcard_ALL) {
-    if (this.happn.utils.wildcardMatch(allPath.replace('/ALL@/', '/*@/'), messageChannel)) {
+    if (this.happn.utils.wildcardMatch(allPath.replace('/ALL@', '/*@'), messageChannel)) {
```

Comment:
this looks like it could be optimized for performance, since the allPath.replace() step could be done once when a listener is added (it would also have to be deleted when the listener is removed)